aes.go

```diff
@@ -9,19 +9,11 @@ import (
 	"errors"
 )
 
-type extraModes interface {
-	// Copied out of crypto/aes/modes.go.
-	NewCBCEncrypter(iv []byte) cipher.BlockMode
-	NewCBCDecrypter(iv []byte) cipher.BlockMode
-	NewCTR(iv []byte) cipher.Stream
-	NewGCM(nonceSize, tagSize int) (cipher.AEAD, error)
```

Comment:
Consider keeping some copied funcs around in interface declarations and use them to "prove" which funcs are implemented by each of the combined structs rather than just commenting which ones are implemented?

Reply:
Done.